packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -219,6 +244,73 @@ def check_statsd_metrics():
             check_statsd_metrics()
 
 
+def test_metrics_public_agent_statsd(dcos_api_session):
```

Comment:
The body of this test looks like a duplicate of `test_metrics_private_agent_statsd()`. Let's either merge those into the same test or extract the duplicated code into a helper function.